doc/service_example.gv

```diff
@@ -0,0 +1,22 @@
+digraph {
```

Comment:
We should regenerate the PNG now.

Reply:
Updated https://github.com/ros2/domain_bridge/pull/26/commits/be8497289da401f3a9e405cc321cd0429cdd1a23